includes/admin/settings/class-wc-settings-products.php

```diff
@@ -473,8 +472,7 @@ public function get_settings( $current_section = '' ) {
 					'default'         => 'yes',
 					'type'            => 'checkbox',
 					'checkboxgroup'   => 'start',
-					'show_if_checked' => 'option',
-					'autoload'        => false
+					'show_if_checked' => 'option'
```

Comment:
Both this and line 326 should gain a trailing comma since we are already changing the lines.